notebooks/official/tabular_workflows/bqml_arima_plus.ipynb

```diff
@@ -703,13 +671,13 @@
         "\n",
         "Now you are ready to start creating your own BigQuery ML ARIMA_PLUS model.\n",
         "\n",
-        "Like with Vertex AI Forecasting, the pipeline you run will train evaluation models using the training and validation sets and use backtesting to create evaluation metrics on the test set. Finally, a serving model will be produced that uses all available data.\n",
+        "Like with Vertex AI Forecasting, the pipeline you run trains evaluation models using the training and validation sets and use backtesting to create evaluation metrics on the test set. Finally, a serving model that uses all available data can be produced.\n",
         "\n",
         "**How do you estimate the cost?**\n",
         "\n",
         "Backtesting involves training a single BigQuery ML model for each period in the test set, so the cost is a function of the length of the test set after any downsampling done by the windowing strategy. The cost is also multiplied by the number of candidate models trained, which is determined by `max_order`.\n",
         "\n",
-        "According to [BQ pricing](https://cloud.google.com/bigquery-ml/pricing), BigQuery ML model creation costs $250 per TB. We'll use a max order of 3, which translates to 20 candidate models when there are multiple time series. Our demo dataset is 3 MB in size, and includes 31 test periods. We window with a stride length of 1, so all periods are used for evaluation.\n",
+        "According to [BQ pricing](https://cloud.google.com/bigquery-ml/pricing), BigQuery ML model creation costs $250 per TB. We can use a max order of 3, which translates to 20 candidate models when there are multiple time series. Our demo dataset is 3 MB in size, and includes 31 test periods. We window with a stride length of 1, so all periods are used for evaluation.\n",
```

Comment:
- Remove "we". Change to "You"
- remove "Our" and change to "The"
- remove "We". Suggested edit (to avoid "we") 
    -     "A stride length of 1 is windowed. All periods are used for evaluation" (to be honest, this doesn't make sense to me.)